### PR TITLE
CLI - created cli scripts for sanity / snap / a11y

### PR DESCRIPTION
**Pull request by danielstro**
## Describe this Pull Request
Currently all scripts are run by using the @ui-autotools/scripts packages. This PR adds scripts to the sanity, snap & a11y packages so that they can be run separately if needed.

### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] None of the above

### Versioning required due to this change
- [ ] Patch
- [x] Minor
- [ ] Major

### Does this change the API or main functionality
- [ ] Yes and I updated README.md
- [ ] No

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -117,7 +117,9 @@ meta.nonEventListenerTestCompliant = true;
 ```
 
 ## CLI Tools
+We have created a package that contains all of the commands to run our tools. If you only want to use some of our packages, you can install each package you want and it comes with its own command.
```
Comment (JoMarton) [inline README.md]: I think we want to direct people to consume the packages separately and deprecate the `scripts` repo soon. So we don't have to put in this paragraph

File: packages/utils/src/cli-utils.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+import { registerRequireHooks } from './';
+import path from 'path';
+
+export function cliInit(): void {
+    dotenv.config();
+    registerRequireHooks();
+}
+
+export const defaultMetaGlob = 'src/**/*.meta.ts?(x)';
+export function getDefaultWebpackConfigPath(projectPath: string): string {
```
Comment (alisey) [inline packages/utils/src/cli-utils.ts]: I'd rename this to `getProjectWebpackConfigPath()`, or simply export `'.autotools/webpack.config.js'`.

File: README.md [hunk 3]
```diff
@@ -139,27 +138,39 @@ Sanity uses puppeteer to test client-side hydration. Results are printed in the
...
 
 #### Usage
-
+First install the package 
+```shell
+npm i --save-dev @ui-autotools/snap
+```
+then run the following command:
 ```shell
 autotools snap --files ./components/**/*.meta.ts
```
Comment (JoMarton) [inline README.md]: `autotools-snap`

File: packages/sanity/src/cli.ts [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+const program = new Command();
+const projectPath = process.cwd();
+const webpackConfigPath = getWebpackConfigPath(projectPath);
+
+program
+    .description('run sanity checks on all components with a metadata description')
+    .option('-f, --files [pattern]', 'Grep file')
+    .action((options) => {
+        const metaGlob: string = options.files || defaultMetaGlob;
+        glob.sync(metaGlob, { absolute: true, cwd: projectPath }).forEach(require);
```
Comment (alisey) [inline packages/sanity/src/cli.ts]: We have an `importMetaFiles()` function in the `scripts` package, use it here or add a TODO comment.

**Pull request merged**